web/services/clusters.go

```diff
@@ -164,7 +164,8 @@ func (s *clustersService) GetAllTags() ([]string, error) {
 	var tags []string
 
 	err := s.db.
-		Model(&models.Tag{ResourceType: models.TagClusterResourceType}).
+		Model(&models.Tag{}).
```

Comment:
Something slipped in the various refactors, this is not working since `Model` only infers the type from the passe struct and ignores conditions. Fixes: #544